pubspec.yaml

```diff
@@ -2,14 +2,27 @@ name: dart_to_js_script_rewriter
 description: Replaces Dart script tags with JavaScript script tags,
              speeding up initial loads and reducing 404s. Use when
              ready to deploy.
-author: Seth Ladd <sethladd@gmail.com>
```

Comment:
Yeah I'd just say something like "Thanks to Seth Ladd for creating the original version of this library" and then link to the original github repo and his email